pubsub.go

```diff
@@ -527,6 +533,13 @@ func WithSeenMessagesTTL(ttl time.Duration) Option {
 	}
 }
 
+func WithAppSpecificRpcInspector(inspector func(peer.ID, *RPC) error) Option {
```

Comment:
I know this is already merged, but we probably want a docstring on this.  Just wanted to flag this 🙂 

cc @yhassanzadeh13

Reply:
Good point.

I am ready to cut a release,but i'll wait for a docstring here.

Reply:
👍 PR incoming.